core/src/main/resources/com/graphhopper/custom_models/racingbike.json

```diff
@@ -1,15 +1,17 @@
 // to use this custom model you need to set the following option in the config.yml
 // graph.elevation.provider: srtm   # enables elevation
-// graph.encoded_values: racingbike_priority, racingbike_access, roundabout, racingbike_average_speed, bike_road_access, average_slope, mtb_rating, hike_rating, sac_scale, country, road_class
+// graph.encoded_values: racingbike_priority, racingbike_access, bike_network, roundabout, racingbike_average_speed, bike_road_access, average_slope, mtb_rating, hike_rating, sac_scale, country, road_class
```

Comment:
@karussell Depending on the answer on my comment to the changes in the file bike.json the same changes might be needed here as well.